tests/testthat/test-conditions.R

```diff
@@ -0,0 +1,17 @@
+context("conditions")
+
+test_that("conditions inherit from `vctrs_error`", {
+  expect_error(stop_incompatible(NULL, NULL), class = "vctrs_error")
```

Comment:
Do you want to test the wording of the error messages here or elsewhere? I've found this useful in tibble, especially for complex messages.

Reply:
The wordings should already be tested elsewhere.